server/graphql/v1/mutations.js

```diff
@@ -515,6 +516,9 @@ const mutations = {
       id: {
         type: new GraphQLNonNull(GraphQLInt),
       },
+      notificationAudience: {
+        type: UpdateAudienceTypeEnum,
```

Comment:
What I understand from the frontend code is that this field will always be set, so let's make it mandatory:

```suggestion
        type: new GraphQLNonNull(UpdateAudienceTypeEnum),
```